src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,31 @@
+.stars_section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  &_reviews {
+    display: flex;
+    align-items: flex-end;
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+  }
+}
+
+.star {
```

Comment:
start looks like separate block, make it elements of starts or move to separate file